client/extensions/wp-super-cache/components/debug/index.jsx

```diff
@@ -68,6 +76,43 @@ class DebugTab extends Component {
 							</FormToggle>
 						</FormFieldset>
 						<div className="wp-super-cache__debug-fieldsets">
+							{ wp_cache_debug_log &&
+								<table>
```

Comment:
What we're trying to display here isn't a table, so we shouldn't be using the `<table>` tag.